sycl/include/CL/sycl/backend.hpp

```diff
@@ -208,6 +208,10 @@ __SYCL_EXPORT device make_device(pi_native_handle NativeHandle,
 __SYCL_EXPORT context make_context(pi_native_handle NativeHandle,
                                    const async_handler &Handler,
                                    backend Backend);
+__SYCL_EXPORT queue make_queue(pi_native_handle NativeHandle,
```

Comment:
Please mark the existing "make_queue" functions as deprecated at least with a comment.. Maybe we have a special macro for that?

Reply:
I have marked it with a comment like similar deprecated/unused APIs.